optaplanner-core/src/main/java/org/optaplanner/core/impl/domain/solution/cloner/gizmo/GizmoSolutionClonerImplementor.java

```diff
@@ -607,7 +629,7 @@ private static void writeDeepCloneMapInstructions(BytecodeCreator bytecodeCreato
             throw new IllegalStateException("Cannot infer element type for Map type (" + type + ").");
         }
 
-        List<Class<?>> entitySubclasses = solutionDescriptor.getSolutionDescriptor().getEntityClassSet().stream()
+        List<Class<?>> entitySubclasses = entityClassOrSubclassList.stream()
                 .filter(keyClass::isAssignableFrom).collect(Collectors.toList());
```

Comment:
I don't understand this formatting choice. Why is `collect()` not on its own line?
(Not the first time in this PR.)